server/.env.example

```
@@ -1,5 +1,6 @@
-ALLOWED_ORIGINS=http://localhost:3000
+ALLOW_ALL_ORIGINS=1
 ALLOW_CORS=1
+ALLOWED_ORIGINS=http://localhost:3001
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ALLOWED_ORIGINS key should go before the ALLOW_ALL_ORIGINS key

(UnorderedKey)